db/seeds.rb

```diff
@@ -8,13 +8,10 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-User.create!(id: 0, first_name: 'Admin', last_name: 'User', email: 'admin@tamu.edu', password: 'T3st!!a')
-Role.create([{ role_name: 'Admin' }, { role_name: 'Tutor' }, { role_name: 'Student' }, { role_name: 'Spartan Tutor' }])
-User.create!(first_name: 'John', last_name: 'Doe', email: 'john@tamu.edu', password: 'T3st!!b')
-user2 = User.create!(first_name: 'Jane', last_name: 'Doe', email: 'jane@tamu.edu', password: 'T3st!!c')
-user3 = User.create!(first_name: 'Jeff', last_name: 'Doe', email: 'jeff@tamu.edu', password: 'T3st!!d')
+admin_role = Role.create(role_name: 'Admin');
+Role.create(role_name: 'tutor');
```

Comment:
Shouldn't either both be capitalized or both be lowercase? This seems like it's asking for errors. Also, why is it assigning to a variable? Where is that going?
```suggestion
admin_role = Role.create(role_name: 'Admin');
Role.create(role_name: 'Tutor');
```

Reply:
Fixed in latest commit